### ISSUE TITLE
InfluxDB 1.x: No error and data are silently ignored in case of invalid time precision

### ISSUE THREAD

**Issue by vlastahajek** [influxdata/influxdb#17622]
__Steps to reproduce:__
Write data with invalid precision constant in the write URL:
```
curl -v -XPOST "http://localhost:8086/write?db=test&precision=x" --data-raw "wifi_status,SSID=AB,device=ESP32 rssi=-31 1586189309000003"

*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8086 (#0)
> POST /write?db=test&precision=x HTTP/1.1
> Host: localhost:8086
> User-Agent: curl/7.55.1
> Accept: */*
> Content-Length: 60
> Content-Type: application/x-www-form-urlencoded
>
* upload completely sent off: 60 out of 60 bytes
< HTTP/1.1 204 No Content
< Content-Type: application/json
< Request-Id: 5516b4bc-7822-11ea-8032-d66d6d2aa003
< X-Influxdb-Build: OSS
< X-Influxdb-Version: 1.8.0rc0
< X-Request-Id: 5516b4bc-7822-11ea-8032-d66d6d2aa003
< Date: Mon, 06 Apr 2020 16:19:04 GMT
```
__Expected behavior:__
Either:
1 - write data
2 - return 500 as it InfluxDB 2 does

__Actual behavior:__
The server returns status as if data is written successfully
`< HTTP/1.1 204 No Content`

__Environment info:__
* System info: Windows 10 Professional
* InfluxDB version: 1.8.0rc1

**Comment by dgnorton** [comment]
Fixed in 1.8.0rc3 by https://github.com/influxdata/influxdb/pull/17656

**Comment by dgnorton** [comment]
Fixed in 1.8.0rc3 by #17656
